realis.go

```diff
@@ -364,16 +379,20 @@ func newClient(realisconfig *RealisConfig) Realis {
 }
 
 // Creates a default Thrift Transport object for communications in gorealis using an HTTP Post Client
-func defaultTTransport(urlstr string, timeoutms int) (thrift.TTransport, error) {
+func defaultTTransport(urlstr string, timeoutms int, secure bool) (thrift.TTransport, error) {
 	jar, err := cookiejar.New(nil)
 	if err != nil {
 		return &thrift.THttpClient{}, errors.Wrap(err, "Error creating Cookie Jar")
 	}
-	transport := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	var transport http.Transport
+	if secure {
+		transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: false}
```

Comment:
Why not just use the "secure" var directly:

```
transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: secure}
```

Reply:
Actually I'll change this code a little bit. Is secure is false I'll modify transport object or else I'll leave the client as it is today